hack/errata.py

```diff
@@ -90,6 +92,58 @@ def notify(message, webhook=None):
     }).encode('utf-8'))
 
 
+def get_open_prs_to_fast(repo):
+    pulls = repo.get_pulls(state='open', sort='created', base='master')
+    for pr in pulls:
+        try:
+            # Skip unknown PRs
+            if not pr.title.startswith("Enable "):
+                continue
+            # Ignore PRs which don't target fast
+            if pr.title.split(" ")[3] != "fast":
+                continue
+            yield pr.number, pr.body
+        except Exception as e:
+            print(f"Failed to parse {pr.number}: {e}")
```

Comment:
My old RHEL still doesn't support f-strings.  Can we use `.format(...)`?

Reply:
Also, we can import `logging` for these warning messages, but I can do that in follow-up work if you prefer.

Reply:
Fixed f-strings in 52ff346fe5975f072c7ca0d2c1e704e7708c9ad3

Reply:
removed remaining f-string in 51a8b4a626c0d33248436ba521be14b2fa5dd976